tests/unit/kernels_reference.hpp

```diff
@@ -30,6 +30,17 @@ namespace reference {
 
     void SGDStep(Parameters& params, float learning_rate);
 
+    void CPUBatchNorm1DForward(const float* input, float* output,
```

Comment:
Why don't you use `Tensor` ? I believe "old" references use raw pointer only because were developed at the moment tensor isn't finished

Reply:
now i use tensor in functions signature, tests became more clear, but reference logic the same, in functions i use raw pointers. Is it ok?

Reply:
You can use whatever, because it will be hidden by the `Tensor`